website/src/pages/tutorial/basic/index.mdx

```diff
@@ -71,3 +71,5 @@ Afterward, you will learn all about the proper error and sanitizing invalid user
 Finally, you'll implement pagination and filtering capabilities to the API!
 
 Let's get started 🚀
+
+> If you wish to add a reaction a comment below, scroll down and make sure to sign into GitHub
```

Comment:
Maybe "If you wish to add a reaction or comment, please scroll down and sign in to GitHub."?